target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1540,14 +1571,19 @@ static int rtl8367b_detect(struct rtl8366_smi *smi)
 			"chip mode");
 		return ret;
 	}
-
-	switch (chip_ver) {
+	if (chip_num == 0x6367 || chip_num == 0x0597 || chip_num == 0x0276) {
+		chip_name = "8367C";
+	} else switch (chip_ver) {
```

Comment:
As suggested by musashino205  I agree we should just use the chip_ver here and ignore the chip_num, but use it for debug output only.
```
case 0x00a0:
	chip_name = "8367C";
	break;
```